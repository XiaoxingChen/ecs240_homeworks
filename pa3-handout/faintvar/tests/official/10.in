c Irreducible loop
p 4 7 10 1 7
c bb1: v1 = v2;
b 1 1 2
e 1 2
e 1 6
c bb2: print(v2, v4);
b 2 0 2 4
e 2 3
e 2 5
e 2 7
c bb3: v2 = c;
b 3 2
e 3 2
e 3 4
c bb4: v4 = v1 + v2;
b 4 4 1 2
e 4 3
c bb5: v2 = c;
b 5 2
e 5 4
c bb6: ;
b 6
e 6 4
c bb7: ;
b 7
