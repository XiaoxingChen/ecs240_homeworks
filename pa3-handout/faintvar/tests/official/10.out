fvin 1 1 3
fvin 2 3
fvin 3 2 3
fvin 4 3 4
fvin 5 2 3 4
fvin 6 3 4
fvin 7 1 2 3 4
